src/detectors/EHCAL/EHCAL.cc

```diff
@@ -62,16 +63,16 @@ extern "C" {
           {
             .readout = "HcalEndcapNHits",
             .fields = {"layer", "slice"},
-            .refs = {1, 0},
+            .refs = {4, 0},
```

Comment:
Can you document the reason for `4` here?

Reply:
This is in order to place the hits in the middle at ~1 interaction length rather than in the front.

Reply:
I meant add a comment in the source code.